### DISCUSSION TITLE
Getting a new error while Model Download

### DISCUSSION THREAD

**Discussion by rstockm** [TheLastBen/fast-stable-diffusion#2000] [Q&A] [answered]
For two days (?) or so, I get this error trying to import a model from my connected google drive - it worked all the time:

<img width="1293" alt="image" src="https://user-images.githubusercontent.com/3195116/232321856-0d61ec2b-86f2-49a3-8ec6-ae7b7bf3830c.png">

Any Ideas?

**Comment by TheLastBen** [comment]
you need to specify if the model is safetensors or not

**Reply by rstockm** [reply]
Interesting. It's a no-safetensor model, so the default "off" should work. But this is what happens:
- load the colab and run with preset unchecked => error
- switch to safetensor on => error
- switch safetensor on and off again (hmmmm...) => works

This is a new behavior, for until now the off-preset just worked.
Nevertheless: up and running so thank you!

**Reply by TheLastBen** [reply, marked as answer]
I implemented a fix, forgot to mention it https://github.com/TheLastBen/fast-stable-diffusion/commit/e5b464dc441df49405eea8816fa789f464a8c8dd
